src/OpenLoco/src/GameCommands/Vehicles/HonkAllTrains.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "GameCommands/GameCommands.h"
+
+namespace OpenLoco::GameCommands
+{
+    struct HonkAllTrainsArgs
```

Comment:
I think it may be better to implement this (and this is on me) as a more generic "honk the horn of the train who's vehicle window is open" and maybe use a -1 or special value to honk all of them?
Perhaps a new button (when new graphics/g1 is done) as well for a button on the vehicle UI to honk it
Though I'd probably wait until commands are reworked for that, its not necessary for this pass